packages/extension/src/providers/kadena/ui/send-transaction/index.vue

```diff
@@ -8,6 +8,14 @@
         </a>
       </div>
 
+      <p
+        class="send-transaction__description"
+        style="color: red"
+        :class="{ popup: isPopup }"
+      >
+        {{ errorMsg }}
+      </p>
+
       <send-address-input
```

Comment:
For the error messaging, can we use a `send-alert` component & the styling that other implementations use?

e.g. ./src/providers/bitcoin/ui/send-transaction/components/send-alert.vue

![1707132493](https://github.com/agencyenterprise/enKrypt/assets/115649719/debddff1-910a-4ab7-80ce-ba76d6700b5a)

Reply:
Hi @Takadenoshi Nice catch!

I made the changes, can you check if it's ok now? Tks

Reply:
Looks good to me!

Reply:
Fixed :)